### ISSUE TITLE
Network Traffic example bugged

### ISSUE THREAD

**Issue by patryk0605** [esnet/react-timeseries-charts#81]
The legend for the [Network Traffic example](http://software.es.net/react-timeseries-charts/#/example/traffic) does not allow you to select both again once you deselect one. Also, it deselects the wrong the data set, for example: clicking on **Into Site** disables **Out of site** instead of disabling **Into Site**.

**Comment by pjm17971** [comment]
Yeah, I noticed that recently as well. The problem is actually that the charts and legend currently support only a single selection, so they work ok for disabling certain items (like the DDoS example), but selection doesn't really work.

Thanks for reporting!

**Comment by pjm17971** [comment]
Additional note: on the legend you can set a `disabled` flag, which actually gives a sort of multiple selection. But the example doesn't work that way, so it needs to be addressed.

**Comment by patryk0605** [comment]
Thanks for the note!

**Comment by patryk0605** [comment]
Quick question regarding the example (a bit off topic), but how is the **G** being rendered on the Y-axis? I am looking at the [source code](https://github.com/esnet/react-timeseries-charts/blob/master/src/website/examples/traffic/Index.js) for the example and do not see where/how it's being added.

**Comment by pjm17971** [comment]
This is from the `format` prop of the YAxis. That prop defaults to showing it like this. (defaultPropType is `format: '.2s'`) But you can specify any format string. They are d3 format strings, so there's lots of examples.

**Comment by patryk0605** [comment]
Great, thanks!
